index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html>
+    <head>
+      <link rel="stylesheet" href="style.css">
+      <link rel="preconnect" href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins:wght@300&display=swap">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+      <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    </head>
+    <body>
+        <header>
+        <nav class="toolbar">
+            <nav class="tbar">
+                <nav class="menu-bar">
+            <img src="John Doe.png" class="john-doe"> 
+            <img src="Icon - Menu.png" class="icon-Menu">
+            </nav>
+        </nav>
+         
+        <nav class="headline">
+            <nav> 
+                <h1>Hey There. I’m Ikoote </h1>
+            <h2>I’am a Software Developer</h2>
+            <p class="txt">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don’t hestiate to contact me.</p>
+            </nav>
+            <img src="Illustration.svg" class="illust" >
```

Comment:
- [ ] Please 🙏🏽 use CSS background if an image is not part of the content (aesthetic reasons) for good practice of HTML&CSS. [Check this link](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image) for more info.If an image is not part of the content (aesthetic reasons) use CSS background-image. [Check this link](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image) for more info.